client/RoomClient.js

```diff
@@ -1,6 +1,6 @@
 import protooClient from './protoo-client/build.js';
 import mediasoupClient from './mediasoup-client/build.js';
-import bowser from'./bowser/src/bowser.js';
+import bowser from './bowser/src/bowser.js';
```

Comment:
Was this file run through a linter or merely copied in.

Since we haven't started linting yet I'd prefer if the original was verbatim.

Reply:
I made these changes to be able to read it, 100s of commented out lines of code. I can revert it and make the minimal changes.

Reply:
Oh I see. Yeah it's better to keep the delta clear, otherwise it will be harder to tell what we actually changed without going through individual commits or deeper code tree understanding tools.

Reply:
okay, I reverted roomclient! The good news is I didn't change any functionality just deleted commented out code.